bchrpc/server.go

```diff
@@ -983,26 +1039,44 @@ func (s *GrpcServer) GetAddressUnspentOutputs(ctx context.Context, req *pb.GetAd
 		}
 	}
 
+	var tokenMetadata []*pb.TokenMetadata
+	if req.IncludeTokenMetadata && s.slpIndex != nil {
+		tokenMetadata = make([]*pb.TokenMetadata, 0)
+		for _hash := range tokenMetadataSet {
+			tm, _ := s.buildTokenMetadata(_hash)
```

Comment:
We should have the error here and log it at the very least.

Reply:
done.